tests/storage/outofprocess_test.py

```diff
@@ -41,6 +41,7 @@
 
```

Comment:
The commit message needs a body explaining this change.

The "Fixes: #NNN" header should be in the end of the commit mesage, before the Signed-off-by: header.